doc/sections/module.md

```diff
@@ -1,45 +1,84 @@
+   This is not yet used,  but will be necessary when we eventually publish our
+   packages online so that they can be automatically downloaded when our users
+   need them. (The idea is to come up with a system similar to Cargo in Rust.)
+ - `root_path` gives the module path under which the library is to be  placed.
+   Assuming that our configuration file is at `<REPO_ROOT>/lambdapi.pkg`, this
+   means that `<REPO_ROOT>/a/b/c.lp` will get module path `<ROOT_PATH>.a.b.c`.
+   In other words this is amounts to giving `--map-dir <ROOT_PATH>:<REPO_ROOT>`
+   on the command line.
+
+In the future,  more useful meta data will be added to the configuration file,
+for example the name of the author, version number, dependencies, ...
+
+Example of configuration file (syntax reference):
 ```
-An important limitation here is the following: if `church.list` depends on the
-`bool` module,  then running `lambdapi` on `list.lp` in the `church` directory
-will fail. Indeed, while in that directory, the module path `bool` corresponds
-to file `church/bool.lp`, which does not exist. Similarly, if `church.list` is
-depending on `church.nat`, then its compilation will also fail due to the fact
-that it expects to find a file `church/church/nat.lp`.
-
-To summarize, every module path in a source tree are relative to the directory
-in which `lambdapi` is invoked. Even for files in (sub-)directories. Note that
-this approach allows us to easily resolve dependencies automatically. However,
-we need to enforce that file paths given on the command line are normalised in
-the following sense. They should not start with `./`, they should be relative,
-and they should not contain `..`.
-```bash
-lambdapi church/nat.lp     # Correct
-lambdapi ./church/nat.lp   # Incorrect
-lambdapi bool.lp           # Correct
-lambdapi church/../bool.lp # Incorrect
+# Lines whose first non-whitespace charater is # are comments  
+# The end of a non-comment line cannot be commented.
+# The config file MUST be called "lambdapi.pkg".
+# The syntax for entries is:
+key = value
+# If the key is not known then it is ignored.
+# There are two used keys for now:
+package_name = my_package_name
+root_path    = a.b.c
+# We will use more entries lated (e.g., authors, version, ...)
 ```
+
+#### Installation procedure for third-party packages
+
+A package should be installed under the library directory, under the specified
+root path. In other words a package whose root path is `x.y.z` should have its
+files installed under the directory `<LIB_ROOT>/x/y/z`. Moreover the directory
+structure relative to the package configuration file should be preserved.
+
+Of course, we should enforce that a root path is uniquely used by one package.
+This can be enforced using a central authority, but this can only be done when
+we actually decide to implement automatic package distribution. For now, it is
+assumed that every user uses reasonable root paths for their packages.
```

Comment:
every user -> everyone
their -> her

Reply:
I believe what I wrote is correct (https://fr.wikipedia.org/wiki/They_singulier).